src/components/footer/footer.hbs

```diff
@@ -15,7 +15,7 @@
             <div class="feedback">
                 <h3 class="feedback__heading">Feedback</h3>
                 <p class="feedback__copy">Please help us to make our accessible experience even better! If you have any questions, comments or feedback, please send us an email</p>
-                <button class="btn btn--black-inverted feedback__submit" type="button" data-modal-show="construction-modal">Submit Feedback</button>
+                <a href="mailto:a11yestoredemo@gmail.com" class="btn btn--black-inverted feedback__submit" role="button">Submit Feedback</a>
```

Comment:
I don't know if we need the `role=button` here, do we?

Reply:
no, I think it should be a link because it is moving away from the page.